starter_code/app.js

```diff
@@ -9,13 +9,36 @@ const punkAPI = new PunkAPIWrapper();
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, 'public')));
 
 
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
+ punkAPI.getBeers()
+   .then(beers => {
+     // res.send(beers)
+     res.render('beers', {beers});
+   })
+   .catch(error => {
+     console.log(error)
+   })
+});
 
+app.get('/random-beer', (req, res, next) => {
+  punkAPI.getRandom()
+    .then(beers => {
+      // res.send(beers)
+      res.render('randomBeer', {beers});
```

Comment:
The API is wrongly coded, it should not return an array of beers containing only one random beer, it should return only the random beer. This problem should be resolved by the server, not by the views. You should render the view like this: `res.render('randomBeer', beers[0]);` and you should not iterate over a beer array in the randomBeer view.